addons/test_mass_mailing/tests/test_mailing_internals.py

```diff
@@ -120,3 +120,21 @@ def test_mailing_trace_utm(self):
             self.assertEqual(test_utm.campaign_id, campaign)
             self.assertEqual(test_utm.source_id, source)
             self.assertEqual(test_utm.medium_id, medium)
+
+    def test_mailing_test_button(self):
```

Comment:
Just to be "that annoying guy": as tests are executed by alphabetical order, better order them in file so that order in logs is the same as order in file.